tools/assess_purity_msms.xml

```diff
@@ -47,87 +42,69 @@
                 --mostIntense
             #end if
 
-            #if $isotopes.isotopes == "exclude_default"
+            #if $AdvOptions.isotopes.isotopes == "exclude_default"
                 --exclude_isotopes
-            #elif $isotopes.isotopes == "user"
+            #elif $AdvOptions.isotopes.isotopes == "user"
                 --exclude_isotopes
-                --isotope_matrix=$isotopes.im
+                --isotope_matrix=$AdvOptions.isotopes.im
             #end if
 
+            @COMMAND_LOG_EXIT@
+
     ]]></command>
+
     <inputs>
 
-        <conditional name="mzML_data">
-            <param name="format" type="select" label="Choose the source for the dataset" >
-                <option value="mzML_files" selected="true">.mzML files to check purity from</option>
-                <option value="library">Library directory name (to be used with dimspy workflows) or path to
-                                        to an individual .mzML file</option>
-            </param>
-            <when value="mzML_files">
-                <param name="source" type="data_collection" format="mzml" label="*.mzML files" >
-                    <validator type="empty_field" />
-                </param>
-            </when>
-            <when value="library">
```

Comment:
@jsaintvanne You need to ask to authors if you can drop the libraries. We did in W4M but I remember that they still use this feature